src/components/OptionListContextProvider.tsx

```diff
@@ -45,6 +51,7 @@ function OptionsListContextProvider({reports, children}: OptionsListProviderProp
     });
 
     const personalDetails = usePersonalDetails();
+    const prevPersonalDetails = useRef(personalDetails);
```

Comment:
can we replace this by our existing hook `usePrevious`?

Reply:
@hungvu193 I tested and we can use `usePrevious` hook. Updated to use this hook.

Reply:
Awesome, let's me test again!